src/v3/plugins/github/urlIdParse.test.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import {
+  issueCommentUrlToId,
+  pullCommentUrlToId,
+  reviewCommentUrlToId,
+  reviewUrlToId,
+} from "./urlIdParse";
+
+describe("plugins/github/urlToId", () => {
+  const issueComment =
+    "https://github.com/example_owner/example-repo0/issues/350#issuecomment-394939349";
```

Comment:
A GitHub username may not actually include an underscore. It’s okay with
me if the RegExp is overly lenient in allowing hyphens (I don’t remember
why we decided to do this back in March), but the test data should
probably be actually valid.

If you want to make the RegExp more strict, that’s okay with me, too.
I am not even opposed to pulling in the `github-username-regex` package
from npm:
https://www.npmjs.com/package/github-username-regex